fetch/api/basic/stream-safe-creation.any.js

```diff
@@ -0,0 +1,33 @@
+// These tests verify that stream creation is not affected by changes to
+// Object.prototype.
+
+const creationCases = {
+  fetch: async () => fetch(location.href),
+  request: () => new Request(location.href),
+  response: () => new Response('hi'),
+  consumeEmpty: () => new Response().text(),
+  consumeNonEmpty: () => new Response(new Uint8Array([64])).text()
+};
+
+for (creationCase of Object.keys(creationCases)) {
+  promise_test(async t => {
+    Object.prototype.start = () => {
```

Comment:
Why make this one throw on call, when .size() throws on get()? It seems like .start shouldn't be gotten by the `CreateReadableStream()` version either.

Reply:
That was indeed not very useful. I replaced it with a version which errors the stream, which is nice because the side-effect is delayed, so if the implementation is (accidentally?) eating exceptions from the constructor, it will still break if it tries to read from the stream later.